_articles/managing-metadata.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Managing Metadata
+layout: article
+excerpt: Merge individual, biospecimen and assay-specific metadata files.
+category: tutorials
+---
+
+File annotations and metadata provide critical information about the data in the AD Knowledge portal. Annotations describe content within the files, including: type of data and assay, tissue of origin, source study, ID of the source individual and type of data. Annotations conform to a structured schema to make data highly searchable and standardized across studies and projects. The standards for defining, managing and implementing controlled vocabularies for content in Synapse is available in the synapseAnnotations Github repository. 
+
+Some types of annotations contain private health information, and cannot be represented as file annotations. For this reason, some of the metadata is stored within a set of csv files. There are typically several metadata csv files required to fully represent details of a single file or individual. 
```

Comment:
> Some types of annotations contain private health information, and cannot be represented as file annotations. For this reason, some of the metadata is stored within a set of csv files.

In AD, this is not the only reason why all metadata is not annotated with. By stating this as being the reason for metadata files existing, it makes it seem like all the data in those metadata files is PHI, which is untrue. I would estimate that most metadata in these files is not PHI, but rather just extra details. People may not want to download the metadata if they think they will be accountable for PHI or if they simply don't believe they need any PHI details for their analysis. Additionally, that first sentence refers to metadata as annotations but then says that they aren't annotations. It's a little confusing.

> There are typically several metadata csv files required to fully represent details of a single file or individual.

While true, this also would need to be used in conjunction with annotations if trying to get the metadata about the file itself. For example, joining the metadata will tell a user who a specimen came from and what assay was done on it, but it won't tell the user which file the resulting assay data is in.

Suggestion:
While a subset of metadata is stored as annotations on the file, the complete metadata is stored within a set of csv files. The complete metadata expands on the annotation set with extra details about the individuals, specimens from the individuals, and the assay performed on the specimens. There are typically several metadata csv files required to fully represent the details of a single individual, specimen, or assay. These can also be combined with the file annotations to get the complete details about the file.